README.md

```diff
@@ -162,6 +162,17 @@ The `AppDelegate.m` file can only have one method for `openUrl`. If you're also
 
 ## Usage
 
+### SDK Initialization
+
+The `autoInitEnabled` option is removed from [facebook-ios-sdk#v9.0.0](https://github.com/facebook/facebook-ios-sdk/blob/master/CHANGELOG.md#900). On iOS, developers are required to initialize the sdk after app launched.
+
+```js
+import { Settings } from 'react-native-fbsdk';
+
+Settings.initializeSDK();
+```
+#### NOTE: This method is also functional on Android
```

Comment:
I would just change the markdown here to use quote instead of h4.
Like this:

> NOTE: This method is also function on Android

Reply:
Sure, quoted text looks better for me.